pyproject.toml

```diff
@@ -97,6 +97,7 @@ sqlalchemy-serializer = "^1.4.22"
 watchdog = "^5.0.3"
 ema-workbench = "^2.5.2"
 names-generator = "^0.2.0"
+ipyparallel = "^8.8.0"
```

Comment:
Is this needed?

Reply:
Well, the `ema-workbench` library complains upon startup if it is not installed, so I installed it here in order to get rid of the startup warning.